bin/tictactoe

```diff
@@ -1,3 +1,13 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+
+# puts "Welcome to Tic Tac Toe!"
+# puts "How many human players?"
+# puts "0, 1, or 2?"
+# game = Game.new(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+# game.play
+
+Game.new.start
```

Comment:
Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.